src/odemis/gui/win/acquisition.py

```diff
@@ -1004,6 +1011,9 @@ def update_setting_display(self):
             area_size_str = util.readable_str(area_size, unit="m", sig=3)
             self.area_size_txt.SetLabel(area_size_str)
 
+            # limit focus point dist based on area size
+            self.focus_points_dist.SetValueRange(50e-6, max(area_size))
```

Comment:
area size is only defined in if and else loop, when if not areas is True, set value range will not work as intended

Reply:
Another point, area size seems to be the addition of all the list of given areas. Should not the range depend on one area at a time ?